cob_twist_controller/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ include_directories(include ${catkin_INCLUDE_DIRS} ${EIGEN_INCLUDE_DIRS} ${oroco
 set(SRC_C_DIR "src/constraint_solvers")
 set(SRC_CS_DIR "${SRC_C_DIR}/solvers")
 
+add_library(jacobian_derivative src/utils/chainjnttojacdotsolver.cpp)
+add_dependencies(jacobian_derivative ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
+target_link_libraries(jacobian_derivative ${catkin_LIBRARIES})
+
```

Comment:
the file `chainjnttojacdotsolver.hpp/cpp` has been take from `orocos_kdl` but it seems not to be integrated in the last ROS released version (1.3.1). This should be removed again and linked against the one in `orocos_kdl` once this is available.